packages/cozy-doctypes/src/contacts/Contact.js

```diff
@@ -9,6 +9,8 @@ const getPrimaryOrFirst = property => obj => {
   return obj[property].find(property => property.primary) || obj[property][0]
 }
 
+const isDeprecated = () => log('warn', 'Deprecated, use contact model in cozy-client instead')
```

Comment:
```suggestion
const isDeprecated = () => log('warn', 'cozy-doctypes contact's model is deprecated, use contact model in cozy-client instead')
```

Reply:
Also, since you're not deprecated all the methods, I'm wondering if we should not be more precise like : 

```
getInitials from cozy-doctypes contact is deprecated. Use cozy-client/models/contacts/getInitials instead
```

More the warning is explicit, more chance we have to take the migration path

Reply:
Done, what do you think ?